ui/app/components/job-template-card.hbs

```diff
@@ -0,0 +1,6 @@
+<Hds::Form::RadioCard data-test-template-card={{@label}} @checked={{@selected}} {{on "change" @onChange}} as |R|>
+    {{#if @icon}}<R.Icon @name={{@icon}} />{{/if}}
+    <R.Label data-test-template-label>{{@label}}</R.Label>
+    {{#if @badge}}<R.Badge @text={{@badge}} />{{/if}}
```

Comment:
Removing this one line should patch our problem in our build process. However, we'll need to address this naming collision challenge later.